context/AnalyticsContext/index.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { createContext, FC, useEffect, useState } from 'react';
```

Comment:
can we wrap all analytics methods bodies inside of try catch statements? just to eliminate any possibility of it breaking functionality which is hardish to test for properly in the deploy preview